packages/core/parcel-bundler/src/utils/ProxyHelper.js

```diff
@@ -0,0 +1,62 @@
+const httpProxy = require('http-proxy');
+const path = require('path');
+const fs = require('fs');
+
+/**
+ * Try to locate the package.json file of the app executing this.
+ * From that packaje.json reads an attribute 'proxy'.
+ *
+ * 'proxy' accepts a string or an object for configuring a proxy service.
+ *
+ * In case that a string is passed, will use it as a host to proxy
+ * and any request to '/api*' will be proxied.
+ *
+ * If an object is passed will lock for 2 keys:
+ * 'proxyHost': the host to proxy the requests
```

Comment:
nit to myself, the documentation is not accurate, the keys are `path` and `host`